protocol/rest/rest_client/resty_client.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package rest_client
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"path"
+	"time"
+)
+
+import (
+	"github.com/go-resty/resty/v2"
+	perrors "github.com/pkg/errors"
+)
+
+import (
+	"github.com/apache/dubbo-go/common/constant"
+	"github.com/apache/dubbo-go/common/extension"
+	"github.com/apache/dubbo-go/protocol/rest/rest_interface"
+)
+
+func init() {
+	extension.SetRestClient(constant.DEFAULT_REST_CLIENT, GetRestyClient)
+}
+
+type RestyClient struct {
+	client *resty.Client
+}
+
+func NewRestyClient(restOption *rest_interface.RestOptions) *RestyClient {
```

Comment:
what's the different between ```NewRestyClient``` and ```GetRestyClient```?

Reply:
`GetRestyClient` will return `rest_interface.RestClient` and `NewRestyClient` will return `*RestyClient`

Reply:
fix